packages/web/global.d.ts

```diff
@@ -7,6 +9,7 @@ declare global {
        * This global is set to true by the prerendering CLI command.
        */
       __REDWOOD__PRERENDERING: boolean
+      __REDWOOD__HELMET_CONTEXT: { helmet?: HelmetData }
```

Comment:
Is this for pre-rendering?

Reply:
Yup - helmet async gathers up all the headers in this variable during prerender - will also be the same if we do this in builder functions in the future.